core-service/src/main/java/org/fundacionjala/sevenwonders/beans/GameMockService.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.fundacionjala.sevenwonders.beans;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.fundacionjala.sevenwonders.core.rest.GameMock;
+import org.springframework.stereotype.Component;
+
+/**
+ *
+ * @author luisgumucio
+ */
+@Component
+public class GameMockService {
```

Comment:
We already talked about GameMockService, it is ok to have a mock service but the contract shouldn't be affected. Please update according our talk.